pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -858,6 +870,87 @@ func registriesConfigIgnition(templateDir string, controllerConfig *mcfgv1.Contr
 	return &registriesIgn, nil
 }
 
+func (ctrl *Controller) createSeccompUseDefaultMC() error {
+	var configMapName = "crio-seccomp-use-default-when-empty"
+
+	// Check if the crio-seccomp-use-default-when-empty config map exists in the openshift-machine-config-operator namespace
+	seccompUseDefaultCM, err := ctrl.kubeClient.CoreV1().ConfigMaps(ctrl.namespace).Get(context.TODO(), configMapName, metav1.GetOptions{})
+	seccompCMIsNotFound := errors.IsNotFound(err)
+	if err != nil && !seccompCMIsNotFound {
+		return fmt.Errorf("error checking for %s config map: %v", configMapName, err)
+	}
+	// If the crio-seccomp-use-default-when-empty config map exists, that means the crio-seccomp-use-default MC was already created
+	// so we should not create this MC again and return
+	if seccompUseDefaultCM != nil && !seccompCMIsNotFound {
+		return nil
+	}
+
+	sel, err := metav1.LabelSelectorAsSelector(metav1.AddLabelToSelector(&metav1.LabelSelector{}, builtInLabelKey, ""))
+	if err != nil {
+		return err
+	}
+	// Find all the MachineConfigPools
+	mcpPoolsAll, err := ctrl.mcpLister.List(sel)
+	if err != nil {
+		return err
+	}
+
+	// Create the crio-seccomp-use-default MC for all the available pools
+	for _, pool := range mcpPoolsAll {
```

Comment:
In hindsight, we probably should just be creating this for master/worker, since all the other pools will inherit from worker anyway, and the user would have to delete 1 MC per pool that is otherwise redundant.

But not a big issue, and shouldn't break anything